burn-book/src/basic-workflow/README.md

```diff
@@ -6,9 +6,17 @@ train a simple convolutional neural network model on the MNIST dataset and prepa
 For clarity, we sometimes omit imports in our code snippets. For more details, please refer to the
 corresponding code in the `examples/guide` [directory](https://github.com/tracel-ai/burn/tree/main/examples/guide).
 We reproduce this example in a step-by-step fashion, from dataset creation to modeling and training
-in the following sections. It is recommended to use the capabilities of your IDE or editor to
+in the following sections. It is recommended to use the capabilities of your IDE or text editor to
 automatically add the missing imports as you add the code snippets to your code.
 
+<div class="warning">
+
+Be sure to checkout the git branch corresponding to the version of Burn you are using to follow
+this guide.
+
+The current version of Burn is `0.14` and the corresponding branch to checkout is `main`.
+</div>
+
```

Comment:
I'm assuming the version for the hosted version will be changed?

Reply:
Yes after the cherry picking.